ui/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/DeploymentService.scala

```diff
@@ -0,0 +1,20 @@
+package pl.touk.nussknacker.engine.api.deployment
+
+import pl.touk.nussknacker.engine.api.ProcessVersion
+
+import scala.concurrent.Future
+
+trait DeploymentService {
+
+  // shouldn't we pass also category as a filter?
+  def getDeployedScenarios(processingType: String): Future[List[DeployedScenarioData]]
```

Comment:
you've right, will fix

Reply:
Can we pass there ProcessingType instead of String?

Reply:
There are two problems with it:
1. `ProcessingType` is in rest model and rest model depends on deployment-manager-api because of some dependencies to `StateStatus` and `ProcessActionType` in returned by manager classes
2. `ProcessingType` is a type inside `pl.touk.nussknacker.restmodel.process` object so can't be moved to other module separately - only with all other classes or with also change of package (object)

Reply:
NVM, this parameter is out after applying comment above